pkg/network/infraconfigurators/bridge.go

```diff
@@ -134,7 +135,9 @@ func (b *BridgePodNetworkConfigurator) PreparePodNetworkInterface() error {
 	if util.IsNonRootVMI(b.vmi) {
 		tapOwner = strconv.Itoa(util.NonRootUID)
 	}
-	err := createAndBindTapToBridge(b.handler, b.tapDeviceName, b.bridgeInterfaceName, b.launcherPID, b.podNicLink.Attrs().MTU, tapOwner, b.vmi)
+
+	queues := converter.CalculateNetworkQueues(b.vmi, converter.GetInterfaceType(b.vmiSpecIface))
+	err := createAndBindTapToBridge(b.handler, b.tapDeviceName, b.bridgeInterfaceName, b.launcherPID, b.podNicLink.Attrs().MTU, tapOwner, queues)
```

Comment:
Can we stop adding attributes to this function ? 

How about you introduce a new wrapping struct - let say `tapDeviceInfo` - and group all this stuff in it ?

Reply:
I replaced one for one, so if refactor is needed lets do it please on a follow PR, not on this one